src/main/java/org/ow2/proactive/addons/ldap_query/LDAPClient.java

```diff
@@ -78,9 +77,10 @@
 
     protected DirContext ldapConnection;
 
-    public LDAPClient(String ldapUrl, String ldapUsername, String ldapPassword, String ldapSearchBase,
-            String ldapSearchFilter, String ldapSelectedAttributes) {
+    public LDAPClient(String ldapUrl, String ldapDnBase, String ldapUsername, String ldapPassword,
```

Comment:
Do we need 2 constructors? why? If we can keep only I would opt for the second one as it has less parameters